kntest/pkg/cluster/gke/README.md

```diff
@@ -0,0 +1,59 @@
+## kntest cluster gke
+
+`kntest cluster gke` command is used for creating, deleting or getting a GKE
+cluster
+
+## Prerequisite
+
+- `GOOGLE_APPLICATION_CREDENTIALS` set
+
+## Usage
+
+This tool can be invoked from command line with following parameters:
+
+- `--min-nodes`: minimum number of nodes, default 1
+- `--max-nodes`: maximum number of nodes, default 3
+- `--node-type`: GCE node type, default "e2-standard-4"
+- `--region`: GKE region, default "us-central1"
+- `--zone`: GKE zone, default empty
+- `--project`: GCP project, default empty
+- `--name`: cluster name, default empty
+- `--release-channel`: GKE release channel, default empty
+- `--resource-type`: Boskos resource type
+- `--version`: GKE version
+- `--backup-regions`: backup regions to be used if cluster creation in primary
+  region failed, comma separated list, default "us-west1,us-east1"
+- `--addons`: GKE addons, comma separated list, default empty
+- `--skip-creation`: should skip creation or not
+
+## Flow
+
+### Create
+
+1. Acquiring cluster if kubeconfig already points to it
+1. Get GCP project name if not provided as a parameter:
+
+   - [In Prow] Acquire from Boskos
+   - [Not in Prow] Read from gcloud config
+
+   Failed obtaining project name will fail the tool
+
+1. Get default cluster name if not provided as a parameter
+1. Delete cluster if cluster with same name and location already exists in GKE
+1. Create cluster
+1. Write cluster metadata to `${ARTIFACT}/metadata.json`
+
+### Delete
+
+1. Acquiring cluster if kubeconfig already points to it
```

Comment:
My understanding of what you wrote, is if kubeconfig points to something, even if the name doesn't match what's passed in, that's the cluster that is deleted.

If that's not the case, you should reword.

Reply:
Done.